src/SpectatorView.Unity/Assets/SpectatorView/Scripts/SpatialAlignment/ISpatialLocalizer.cs

```diff
@@ -1,26 +1,27 @@
-﻿using Microsoft.MixedReality.Experimental.SpatialAlignment.Common;
-using Microsoft.MixedReality.Toolkit.Extensions.Experimental.Socketer;
-using System;
-using System.Collections;
-using System.Collections.Generic;
-using System.IO;
-using System.Threading;
-using System.Threading.Tasks;
-using UnityEngine;
-
-namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView
-{
-    public interface IPeerConnection
-    {
-        void SendData(Action<BinaryWriter> writeCallback);
-    }
-
-    public interface ISpatialLocalizer
-    {
-        Guid SpatialLocalizerId { get; }
-
-        bool TryDeserializeSettings(BinaryReader reader, out ISpatialLocalizationSettings settings);
-
-        bool TryCreateLocalizationSession(IPeerConnection peerConnection, ISpatialLocalizationSettings settings, out ISpatialLocalizationSession session);
-    }
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.SpectatorView
+{
+    public interface IPeerConnection
+    {
+        void SendData(Action<BinaryWriter> writeCallback);
+    }
+
+    public interface ISpatialLocalizer
```

Comment:
>ISpatialLocalizer [](start = 21, length = 17)

Do we know whether we want ISpatialLocalizer and ISpatialLocalizationSettings to be maintained in the SpatialAlignment repo/logic?

Reply:
The current plan is to move this code as part of a future task.